website/docs/docs/build/derived-metrics.md

```diff
@@ -21,7 +21,8 @@ In MetricFlow, derived metrics are metrics created by defining an expression usi
 | `metrics` |  The list of metrics used in the derived metrics. | Required  |
 | `alias` | Optional alias for the metric that you can use in the expr. | Optional |
 | `filter` | Optional filter to apply to the metric. | Optional |
-| `offset_window` | Set the period for the offset window, such as 1 month. This will return the value of the metric one month from the metric time. | Required |
+| `offset_window` | Set the period for the offset window, such as 1 month. This will return the value of the metric one month from the metric time. This can't be used with `offset_to_grain`. | Required |
+| `offset_to_grain` | Specifies the granularity or level of detail for the offset, such as a day. This means if you set your `offset_to_grain: day`, the offset is applied at the daily level. If you set it to "hour," it means the offset is at the hourly level. This can't be used with `offset_window`. | Required |
```

Comment:
add to table and also clarified they can't be used together [per pr here](https://github.com/dbt-labs/metricflow/pull/744/files#diff-ef677cc7ad90e1061403e18f214a05fcffd11419f99d094c6bce47be9934e31eL737) which states that. can you verify @courtneyholcomb ?

Reply:
Can we take out `offset_to_grain` from the docs for now? I'll write up some example use cases when I'm back from OOO and we can add it back. 

I think this option was added to satisfy a really specific use case from a customer at Transform. We need to do a bit more testing internally to figure out the right situations to use this option before we document it.